tests/safeds/data/tabular/containers/_column/test_plot_boxplot.py

```diff
@@ -27,3 +30,12 @@ def test_plot_boxplot_int(monkeypatch: _pytest.monkeypatch) -> None:
     monkeypatch.setattr(plt, "show", lambda: None)
     table = Table.from_dict({"A": [1, 2, 3]})
     table.get_column("A").plot_boxplot()
+
+
+def test_plot_boxplot_legacy() -> None:
+    table = Table.from_dict({"A": [1, 2, 3]})
+    # table.get_column("A").plot_boxplot().to_png_file(resolve_resource_path("./image/snapshot_boxplot.png"))
```

Comment:
This comment should be removed (same for other commented-out code). We can easily call `to_png_file` on `current` in the future if necessary.